### ISSUE TITLE
Accessibility: carousel buttons have no discernible text

### ISSUE THREAD

**Issue by dgodongm** [mastodon/joinmastodon#361]
The buttons for navigating the carousels for the "Why Mastodon?" and "What our users are saying" content do not have any discernible text.

Repro Steps:
With screenreader on
1. Navigate to https://joinmastodon.org/
2. Tab to "What our users are saying" carousel button and then tab to next one

Results:
Screenreader is only able to indicate that element is a button. There's not indication of what it does or that it controls navigation of the carousel

Alternate repro: Use Axe devtools, WAVE, or other A11N tools

Expected Results:
Buttons should have aria-label set to something like "Go to slide <x>"

There are arguably further a11y/usability improvements that could be done to improve the carousel, but the above seems like minimal to ensure a11y. W3C’s WAI-ARIA working group provides further guidance at https://www.w3.org/WAI/ARIA/apg/patterns/carousel/

**Comment by dgodongm** [comment]
Other helpful guidance on making carousel's more accessible: 
https://www.smashingmagazine.com/2023/02/guide-building-accessible-carousels/
https://dev.to/jasonwebb/how-to-build-a-more-accessible-carousel-or-slider-35lp
